### ISSUE TITLE
customization of learning rate

### ISSUE THREAD

**Issue by tkng** [blue-oil/blueoil#3]
Currently, Blueoil doesn't support customization of learning rate. It would be very nice if we can customize learning rate, since it is a very important hyperparameter.

For example, internally, we've used some learning rate scheduling, like `tf.train.piecewise_constant`, `tf.train.polynomial_decay`.
